diff --git a/node_modules/beachball/lib/packageManager/npmArgs.js b/node_modules/beachball/lib/packageManager/npmArgs.js
index 0965cff..005b750 100644
--- a/node_modules/beachball/lib/packageManager/npmArgs.js
+++ b/node_modules/beachball/lib/packageManager/npmArgs.js
@@ -9,7 +9,9 @@ exports.getNpmLogLevelArgs = getNpmLogLevelArgs;
 function getNpmPublishArgs(packageInfo, options) {
     const { registry, access } = options;
     const args = [
-        'publish',
++       'run',
++       'invoke-publish',
++       '--',
         '--registry',
         registry,
         '--tag',
