diff --git a/node_modules/beachball/lib/packageManager/npmArgs.js b/node_modules/beachball/lib/packageManager/npmArgs.js
index 022b51d..171c4b7 100644
--- a/node_modules/beachball/lib/packageManager/npmArgs.js
+++ b/node_modules/beachball/lib/packageManager/npmArgs.js
@@ -10,7 +10,9 @@ function getNpmPublishArgs(packageInfo, options) {
     const { registry, access } = options;
     const authArgs = getNpmAuthArgs(options);
     const args = [
-        'publish',
+        'run',
+        'invoke-publish',
+        '--',
         '--registry',
         registry,
         '--tag',
