diff --git a/node_modules/beachball/lib/packageManager/npmArgs.js b/node_modules/beachball/lib/packageManager/npmArgs.js
index 60dbbc6..8696cbe 100644
--- a/node_modules/beachball/lib/packageManager/npmArgs.js
+++ b/node_modules/beachball/lib/packageManager/npmArgs.js
@@ -10,8 +10,9 @@ function getNpmPublishArgs(packageInfo, options) {
     const pkgCombinedOptions = packageInfo.combinedOptions;
     const authArgs = getNpmAuthArgs(options);
     const args = [
-        'publish',
-        '--registry',
+        'run',
+        'invoke-publish',
+        '--',
         registry,
         '--tag',
         pkgCombinedOptions.tag || pkgCombinedOptions.defaultNpmTag || 'latest',
